spec/forms/prior_authority/send_back_form_spec.rb

```diff
@@ -46,4 +46,73 @@
       end
     end
   end
+
+  describe '#save' do
+    let(:fixed_arbitrary_date) { DateTime.new(2023, 12, 3, 12, 3, 12) }
+    let(:user) { create(:caseworker) }
+
+    before do
+      allow(NotifyAppStore).to receive(:process)
+      travel_to fixed_arbitrary_date
+      create(:assignment, submission:, user:)
+    end
+
+    context 'when params are invalid' do
+      let(:params) do
+        {
+          updates_needed: ['further_information'],
+          further_information_explanation: '',
+          submission: submission
+        }
+      end
+
+      it 'returns false' do
+        expect(subject.save).to be false
```

Comment:
**nit and outside scope of work**, and i guess we do not have rspec rubocop in or configured, but i like the cop and logic that encourages named subjects - `form.save` is much more readable but either way